core/src/test/resources/sql/oracle-func.iq

```diff
@@ -0,0 +1,49 @@
+# oracle-func.iq - Queries involving Oracle Functions
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+!use oraclefunc
+!set outputformat mysql
+
+# XML Functions
+
+SELECT XMLTRANSFORM(
+                   '<?xml version="1.0"?>
+                    <Article>
+                      <Title>My Article</Title>
+                      <Authors>
+                        <Author>Mr. Foo</Author>
+                        <Author>Mr. Bar</Author>
+                      </Authors>
+                      <Body>This is my article text.</Body>
+                    </Article>'
+                   ,
+                   '<?xml version="1.0"?><xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+                      <xsl:output method="text"/><xsl:template match="/">Article - <xsl:value-of select="/Article/Title"/>Authors: <xsl:apply-templates select="/Article/Authors/Author"/>
+                      </xsl:template>
+                      <xsl:template match="Author">- <xsl:value-of select="." /></xsl:template>
+                    </xsl:stylesheet>'
+                   );
++-------------------------------------------------+
+| EXPR$0                                          |
++-------------------------------------------------+
+| Article - My ArticleAuthors: - Mr. Foo- Mr. Bar |
++-------------------------------------------------+
+(1 row)
+
```

Comment:
Can we merge this file with `mysql-func.iq` and name it xml-func.iq ? Because each test file only contains one query and they are all xml functions test.

Reply:
These files were kept separate intentionally so that existing and new tests (for oracle, mysql, etc) can be moved to respective files and also to keep test files modular

Reply:
There is already a `functions.iq`, maybe we can move all the function tests there.

Reply:
look at all the existing .iq files, it seems that we never create a new file for a sql dialect. And i think that is too much to maintain for us.

Reply:
functions.iq belongs to plus sub project, Not sure if it is the right place to add these test cases. Creating xml-func.iq files as mentioned I feel is still too much to maintain, Keeping functions per dialact seems more manageable to me, If you still feel all the functions should be  within functions.iq in core, I will raise another PR :)

Reply:
no, just in this PR.

Reply:
Done